lcov.info

```diff
@@ -0,0 +1,441 @@
+SF:synapse_invite_checker/__init__.py
```

Comment:
Do we really need this file or is this a mistake? This file was not on the other PRs so I'm not sure about it

Reply:
no, this should be ignored, i just missed it. thank you